Ownership/TC - LTC/London TC NE.txt

```diff
@@ -8,6 +8,18 @@ ALTOWNER:Observing LON_C_CTR:TCGS:TCNE:TCN:TC:TCE:LEN:LE:LGS:O:LM:LC:LNC:LWC:LSC
 ALTOWNER:Observing LON_SC_CTR:TCGS:TCNE:TCN:TC:TCE:LEN:LE:LC:LNC:LWC:LGS:O:LM:LSC:L
 ALTOWNER:Observing London FIR:TCGS:LGS:O:LM:TCNE:TCN:TC:TCE:LEN:LE:LC:LNC:LWC:LSC:L
 
+SECTOR:SODVU Box:8500:10500
+OWNER:TCSE:TCS:TC:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LON_SC_CTR:TCGS:TCSE:TCS:TC:LD:LS:LWS:LGS:O:LM:LSC:L
+ALTOWNER:Observing LON_S_CTR:TCGS:TCSE:TCS:TC:LD:LGS:O:LM:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_S_CTR:TCGS:TCSE:LGS:O:LM:TCS:TC:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LTC_CTR:TCGS:TCSE:TCS:LGS:O:LM:TC:LD:LS:LWS:LSC:L
+ALTOWNER:Observing LON_D_CTR:TCGS:TCSE:TCS:TC:LGS:O:LM:LD:LS:LWS:LSC:L
+ALTOWNER:Observing London FIR:TCGS:LGS:O:LM:TCSE:TCS:TC:LD:LS:LWS:LSC:L
+BORDER:SODBU Box
```

Comment:
Should this line be
`BORDER:SODVU Box`

Might be wrong, first review!

Reply:
Yes, indeed! I had fixed it locally (otherwise I wouldn't have been able to get that screenshot) but clearly didn't update on the branch. Thanks!